src/rules/checkLineAlignment.js

```diff
@@ -97,41 +103,93 @@ const checkNotAlignedPerTag = (utils, tag) => {
   utils.reportJSDoc('Expected JSDoc block lines to not be aligned.', tag, fix, true);
 };
 
-const checkAlignment = ({
+const filterApplicableTags = (jsdoc, applicableTags) => {
```

Comment:
I'm not sure if it's a good approach with comment-parser, but it's working. Basically, we remove the tags we don't want to align to align only the ones we want.